src/nashpy/game.py

```diff
@@ -423,3 +424,62 @@ def linear_program(self):
         row_strategy = linear_program(row_player_payoff_matrix=A)
         column_strategy = linear_program(row_player_payoff_matrix=B.T)
         return row_strategy, column_strategy
+
+    def regret_minimization(self, learning_rate=0.1, max_iterations=100):
```

Comment:
```suggestion
    def regret_minimization(self, learning_rate=0.1, iterations=100):
```

Reply:
Changed the max_iterations to iterations in all the codes and documents